cmd/talosctl/cmd/talos/reboot.go

```diff
@@ -31,5 +43,6 @@ var rebootCmd = &cobra.Command{
 }
 
 func init() {
+	rebootCmd.Flags().IntP("mode", "m", 0, "skips kexec and reboots with power cycle")
```

Comment:
I think `int` mode is super confusing. We could instead use `string` flag with a set of accepted values:

* `default` 
* `powercycle`